### PR TITLE
Improve visualisations and update report

### LINKED ISSUE #18: Remove `facet_wrap()` from figures in report
Currently we plot one panel for each subgroup. I think it would be clearer to have all subgroups in one figure, each visualised in a different colour/shape, e.g., like [this example](https://openheart.bmj.com/content/openhrt/11/2/e002732/F4.large.jpg).

As part of this we can also consider making some improvements to code [plot_measures()](https://github.com/opensafely/pharmacy-first/blob/b0137c7ae756ae1d238420663d3ea48990323ad4/lib/functions/function_plot_measures.R#L17C1-L116) function, but this could also come later. Things to consider are (1) reducing the number of arguments that are needed and (2) finding a less verbose way to define the dictionaries for the labels, see [here](https://github.com/opensafely/pharmacy-first/blob/b0137c7ae756ae1d238420663d3ea48990323ad4/reports/pharmacy_first_report.Rmd#L26-L121).

### PR DESCRIPTION
**Pull request by milanwiedemann**
Closes #18

This may require some changes because of updates in #19

### CODE REVIEW

File: reports/pharmacy_first_report.Rmd [hunk 1]
```diff
@@ -142,68 +91,103 @@ Links to the codelist for each analysis can be found beneath the relevant sectio
...
+```{r, message=FALSE, warning=FALSE, fig.height=8, fig.width=8}
+# Select measures and breakdown
+df_measures_selected <- df_measures %>%
+  filter(measure_desc == "clinical_service") %>%
+  filter(group_by == "Age")
+
+# Create visualisation
+plot_measures(
+  df_measures_selected,
+  select_value = denominator,
```
Comment (viv3ckj) [inline reports/pharmacy_first_report.Rmd:125]: This should be numerator instead, right? If so, same applies for all the others.

File: lib/functions/function_tidy_measures.R [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#' Tidy measures data
+#'
+#' Creates a tidier dataframe of measures data.
+#' The measures must be named in a spedific way for this function to work properly.
+#'
+#' @param data A dataframe containing output from the OpenSAFELY measures framework
+#' @param pf_measures_name_mapping List, specifying the dict of measure names.
```
Comment (viv3ckj) [inline lib/functions/function_tidy_measures.R:7]: param pf_measures_name_dict List, specifying the dict of measure names

File: lib/functions/function_plot_measures.R [hunk 3]
```diff
@@ -2,94 +2,66 @@
...
 #' @return A ggplot object.
 
 plot_measures <- function(
     data,
-    measure_names,
-    custom_labels = NULL,
-    date_col = "interval_end",
-    value_col = "numerator",
-    measure_col = "measure",
+    select_measures = NULL,
```
Comment (viv3ckj) [inline lib/functions/function_plot_measures.R:23]: Will select_measures vary, or will it always be numerator? If so, might as well hardcode it?

**Review by viv3ckj — CHANGES_REQUESTED** [review]

File: reports/pharmacy_first_report.Rmd (hunk 1 above)
Comment (milanwiedemann) [reply reports/pharmacy_first_report.Rmd:125]: Ohhh yes, you're right! Good spot

File: lib/functions/function_tidy_measures.R (hunk 2 above)
Comment (milanwiedemann) [reply lib/functions/function_tidy_measures.R:7]: thanks :)

Comment (milanwiedemann) [reply lib/functions/function_tidy_measures.R]: addressed in a82ca2f46270574a6c65067f0e332b908e8a9cab

File: lib/functions/function_plot_measures.R (hunk 3 above)
Comment (milanwiedemann) [reply lib/functions/function_plot_measures.R]: I think you're right, have removed this in 187b1579907882686856d6696d41f0a73735f526

File: reports/pharmacy_first_report.Rmd (hunk 1 above)
Comment (milanwiedemann) [reply reports/pharmacy_first_report.Rmd]: all changed to numerator in 7b19a7c5659e54cb9bff33ea051a6d716c789258

**Review by viv3ckj — APPROVED** [review]

**Pull request merged**